avaframe/com1DFA/com1DFA.py

```diff
@@ -644,17 +641,20 @@ def createReportDict(avaDir, logName, relName, inputSimLines, cfgGen, reportArea
     resistanceArea = inputSimLines['resistanceArea']
     relDict = inputSimLines['releaseLine']
 
+
+    # Get default cfg and convert to dict for comparison
+    cfgGen = cfg['GENERAL']
+
     # Create dictionary
     reportST = {}
-    reportST = {}
     reportST = {'headerLine': {'type': 'title', 'title': 'com1DFA Simulation'},
                 'avaName': {'type': 'avaName', 'name': str(avaDir)},
                 'simName': {'type': 'simName', 'name': logName},
                 'time': {'type': 'time', 'time': dateTimeInfo},
                 'Simulation Parameters': {
                 'type': 'list',
                 'Program version': getVersion(),
-                'Parameter set': '',
+                'Parameter set': 'Default',
                 'Release Area Scenario': relName,
                 'Entrainment': entInfo,
                 'Resistance': resInfo,
```

Comment:
remove the next two keys: Parameter variation on and parameter value as the changed parameters are now anyway listed separately